com.unity.render-pipelines.high-definition/Runtime/Debug/DebugDisplay.cs

```diff
@@ -1288,6 +1290,58 @@ void RegisterLightingDebug()
                     });
                 }
 
+                if (HDRenderPipeline.allowDebugMaterialSamplerOverride)
```

Comment:
You should put this control in the Material section of the debug menu and not in the lighting section

Reply:
will do

Reply:
addressed